cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -247,106 +247,56 @@ spec:
     app: {{ template "kubecost.clusterControllerName" . }}
 ---
 # TurndownSchedule Custom Resource Definition for persistence
-{{- if .Capabilities.APIVersions.Has "apiextensions.k8s.io/v1" }}
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: turndownschedules.kubecost.k8s.io
+  name: turndownschedules.kubecost.com
   labels:
     {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
 spec:
-  group: kubecost.k8s.io
+  group: kubecost.com
+  names:
+    kind: TurndownSchedule
+    singular: turndownschedule
+    plural: turndownschedules
+    shortNames:
+    - td
+    - tds
+  scope: Cluster
   versions:
     - name: v1alpha1
       served: true
       storage: true
+      subresources:
+        status: {}
       schema:
         openAPIV3Schema:
+          type: object
           properties:
             spec:
               type: object
               properties:
-                start: 
+                start:
                   type: string
                   format: date-time
                 end:
                   type: string
                   format: date-time
-                repeat: 
+                repeat:
                   type: string
                   enum: [none, daily, weekly]
-      subresources:
-        status: {}
       additionalPrinterColumns:
-        - name: State
-          type: string
-          description: The state of the turndownschedule 
-          jsonPath: .status.state
-        - name: Next Turndown
-          type: string
-          description: The next turndown date-time
-          jsonPath: .status.nextScaleDownTime
-        - name: Next Turn Up
-          type: string
-          description: The next turn up date-time
-          jsonPath: .status.nextScaleUpTime
-  names:
-    kind: TurndownSchedule
-    singular: turndownschedule
-    plural: turndownschedules
-    shortNames:
-    - td
-    - tds
-  scope: Cluster
-{{ else }}
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Do we have a min k8s version we support/guarantee to run on? 1.16 was the first version that beta was added.